docs/reference/modules/cluster/remote-clusters-connect.asciidoc

```diff
@@ -0,0 +1,219 @@
+[[remote-clusters-connect]]
+=== Connect to remote clusters
+Your local cluster uses the <<modules-network,transport layer>> to establish
```

Comment:
```suggestion
Your local cluster uses the <<modules-network,transport interface>> to establish
```

The linked networking doc uses the phrase "transport interface", which I like better for clarity than either "transport layer" or "transport level". I think we should strive for consistency here.